widgetssdk/src/main/java/com/glia/widgets/core/dialog/Dialog.java

```diff
@@ -21,11 +21,12 @@ public final class Dialog {
     public static final int MODE_VISITOR_CODE = 11;
     public static final int MODE_MESSAGE_CENTER_UNAVAILABLE = 12;
     public static final int MODE_UNAUTHENTICATED = 13;
+    public static final int MODE_LIVE_OBSERVATION_OPT_IN = 14;
 
     @IntDef({MODE_NONE, MODE_OVERLAY_PERMISSION, MODE_UNEXPECTED_ERROR, MODE_NO_MORE_OPERATORS, MODE_ENGAGEMENT_ENDED,
-            MODE_EXIT_QUEUE, MODE_START_SCREEN_SHARING, MODE_END_ENGAGEMENT, MODE_ENABLE_NOTIFICATION_CHANNEL,
-            MODE_ENABLE_SCREEN_SHARING_NOTIFICATIONS_AND_START_SHARING, MODE_MEDIA_UPGRADE, MODE_VISITOR_CODE,
-            MODE_MESSAGE_CENTER_UNAVAILABLE, MODE_UNAUTHENTICATED})
+        MODE_EXIT_QUEUE, MODE_START_SCREEN_SHARING, MODE_END_ENGAGEMENT, MODE_ENABLE_NOTIFICATION_CHANNEL,
+        MODE_ENABLE_SCREEN_SHARING_NOTIFICATIONS_AND_START_SHARING, MODE_MEDIA_UPGRADE, MODE_VISITOR_CODE,
+        MODE_MESSAGE_CENTER_UNAVAILABLE, MODE_UNAUTHENTICATED, MODE_LIVE_OBSERVATION_OPT_IN})
```

Comment:
Are we expecting a lot of Live observation dialogs? If not then this could just be MODE_LIVE_OBSERVATION for simplicity sake.

Reply:
It's likely that this is the only dialog related to LO, but I want to keep it as is because `MODE_LIVE_OBSERVATION` doesn't accurately describe the type of dialog.